Meta/.shell_include.sh

```diff
@@ -13,3 +13,32 @@ die() {
     echo "die: $*"
     exit 1
 }
+
+find_executable() {
+  paths=("/usr/sbin" "/sbin")
+
+  if [ "$(uname -s)" = "Darwin" ]; then
+    paths+=("/usr/local/opt/e2fsprogs/bin" "/usr/local/opt/e2fsprogs/sbin")
+    paths+=("/opt/homebrew/opt/e2fsprogs/bin" "/opt/homebrew/opt/e2fsprogs/sbin")
+  fi
```

Comment:
Eh, I would have copied the block to the bottom of the file as-is, but I guess this works as well.

Reply:
I hope so :eyes:

Reply:
The block of code that this is supposed to replace is still there though. :^)